grakn-graph/src/main/java/ai/grakn/factory/SystemKeyspace.java

```diff
@@ -186,8 +198,8 @@ public static void deleteKeyspace(String keyspace){
      * only consists of types, the inserts are idempotent and it is safe to load it
      * multiple times.
      */
-    private static void loadSystemOntology() {
-        try (GraknGraph graph = factory().open(GraknTxType.WRITE)) {
+    private static void loadSystemOntology(InternalFactory factory) {
```

Comment:
If you use the `factory()` method there is a circular dependency - you will be waiting for the countdown latch to release forever, because you are now blocking the thread that should release it.